contrib/win32/win32compat/console.c

```diff
@@ -192,6 +192,10 @@ ConEnterRawMode()
 	 *  so that the clearscreen will not erase any lines.
 	 */
 	if (TRUE == isAnsiParsingRequired) {
+		if (gcsbRet == 0)
+		{
+			error("Failed to get console screen buffer info error:%d", GetLastError());
```

Comment:
need to return from here or continue with line 199?

Reply:
modified to return from here